rust/tools/authorization-logic/cargo/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+"""
+@generated
+cargo-raze generated Bazel file.
+
+DO NOT EDIT! Replaced on runs of cargo-raze
+"""
```

Comment:
Not a fan in general of checking in generated files that have to be manually regenerated; it is asking for things to go out of sync. I gather it's hard to do otherwise with Bazel's model, though, and it currently looks like the Rust side of this project isn't long for this world. I don't know that there's anything to be done here, just wish there was a better way.

Reply:
Agree with the sentiment about not checking in generated files. I did consider a few options before creating the PR:
 - Configure `cargo raze` to be runnable with a bazel  ([using-cargo-raze-through-bazel](https://github.com/google/cargo-raze#using-cargo-raze-through-bazel))
 - Define a bazel or macro that invokes `cargo raze`. 
 - Make the above macro/bazel rule to be a dependency of `authorization_logic` target. 
It required some additional work. Therefore, I decided to defer it to later if this becomes an issue.